.github/workflows/docker-image.yml

```diff
@@ -22,8 +22,14 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           
-      - name: building docker image
-        run: docker build -t elimuinformatics/a2d2:${GITHUB_REF##*/} -t elimuinformatics/a2d2:latest .
-
-      - name: Publish the Docker image
-        run: docker push . --tag elimuinformatics/a2d2:${GITHUB_REF##*/} --tag elimuinformatics/a2d2:latest
+      - name: build image
+        run: docker build -t elimuinformatics/a2d2:latest .
+      
+      - name: tag image with commit id
+        run: docker image tag elimuinformatics/a2d2:latest elimuinformatics/a2d2:${GITHUB_REF##*/} 
```

Comment:
We do NOT want to tag it with the commit id. The tags should be `latest` and a version number.

Reply:
@hankwallace By mistake it was commit Id, It should be version id. So I have updated the PR

Reply:
What is "version id"?

Reply:
In order to trigger github action we need to create a tag ID, which will be used as docker image version id.